### DISCUSSION TITLE
v1.10.17-lts

### DISCUSSION THREAD

**Discussion by wanghe-fit2cloud** [1Panel-dev/1Panel#6578] [Announcements]
## 问题修复

* 【系统】修复了在暗色模式下页面底部出现白色横向滚动条的问题；
* 【系统】修复了概览页底部未对齐的问题；
* 【系统】修复了 WAF 和网站监控概览页状态栏显示错位的问题。

<hr /><em>This discussion was created from the release <a href='https://github.com/1Panel-dev/1Panel/releases/tag/v1.10.17-lts'>v1.10.17-lts</a>.</em>

**Comment by pyzhf3yj** [comment]
概览页面能显示温度就好了
